app/models/opaque_token.rb

```diff
@@ -24,12 +25,16 @@ def set_aud(aud)
     @aud = aud
   end
 
+  def set_ssoi(ssoi)
+    @ssoi = ssoi
```

Comment:
No linter, renamed to `is_ssoi` to more clearly indicate it is a boolean.

In a future PR I plan to add a token type enum that can be saved as part of the session rather than these one-off fields/methods.